src/header/header.scss

```diff
@@ -257,10 +260,16 @@
     }
   }
 
-  > .iui-icon {
+  > .iui-button-icon {
     flex-shrink: 0;
   }
 
+  > .iui-button-icon + .iui-button-label,
+  > .iui-button-label + .iui-button-icon,
+  > .iui-button-icon + .iui-button-icon {
+    margin-left: $iui-sm;
```

Comment:
Why add this back? / why unset the gap?

Reply:
Because left gap does not disappear when button is getting smaller on smaller screens:
![image](https://user-images.githubusercontent.com/83585998/142592400-2ae21b3f-9b33-4958-9dbc-73d3ebe1b1ce.png)
![image](https://user-images.githubusercontent.com/83585998/142592438-07fedc49-cf9d-4790-9961-9064363f0a3f.png)

Reply:
In that case, can we at least change the selector? We don't need those specific classes.
```suggestion
  > * + * {
    margin-left: $iui-sm;
```